config/config.go

```diff
@@ -290,6 +291,7 @@ func New() *Config {
 		Prometheus: Prometheus{
 			ExternalURLRaw: "",
 			PageTitle:      "Prometheus Time Series Collection and Processing Server",
+			Listen:         ":9090",
```

Comment:
Possibly conflict with main http listener ?

Reply:
yep. changed to 9092